lua/nvim-tree/explorer/sorters.lua

```diff
@@ -116,8 +128,10 @@ function M.setup(opts)
   M.sort_by = opts.sort_by
   if M.sort_by == "modification_time" then
     M.node_comparator = M.node_comparator_modification_time
+  elseif M.sort_by == "Name" then
```

Comment:
name should be the default; the else

Reply:
nae is still the default. 'Name' is for case sensitive.